test/test_fp8.py

```diff
@@ -0,0 +1,51 @@
+import os
```

Comment:
should you also add this test to `test/tpu/run_test.sh`?

Reply:
Added